esda/tests/test_geary.py

```diff
@@ -27,32 +27,33 @@ def test_Geary(self):
         self.assertAlmostEqual(c.z_norm, -3.7399778367629564)
         self.assertAlmostEqual(c.seC_norm, 0.17834040553138225)
 
-        self.assertAlmostEqual(c.VC_rand, 0.018437747611029367)
-        self.assertAlmostEqual(c.p_rand, 4.5059156794646782e-07)
-        self.assertAlmostEqual(c.z_rand, -4.9120733751216008)
-        self.assertAlmostEqual(c.seC_rand, 0.13578566791465646)
+        self.assertAlmostEquals(c.VC_rand,0.033411917666958356)
+        self.assertAlmostEquals(c.p_rand,0.00013165646189214729)
+        self.assertAlmostEquals(c.z_rand, -3.6489513837253944)
+        self.assertAlmostEquals(c.seC_rand, 0.18278927120309429)
 
         np.random.seed(12345)
         c = geary.Geary(self.y, self.w, permutations=999)
-        self.assertAlmostEqual(c.C, 0.33301083591331254)
-        self.assertAlmostEqual(c.EC, 1.0)
-
-        self.assertAlmostEqual(c.VC_norm, 0.031805300245097874)
-        self.assertAlmostEqual(c.p_norm, 9.2018240680169505e-05)
-        self.assertAlmostEqual(c.z_norm, -3.7399778367629564)
-        self.assertAlmostEqual(c.seC_norm, 0.17834040553138225)
-
-        self.assertAlmostEqual(c.VC_rand, 0.018437747611029367)
-        self.assertAlmostEqual(c.p_rand, 4.5059156794646782e-07)
-        self.assertAlmostEqual(c.z_rand, -4.9120733751216008)
-        self.assertAlmostEqual(c.seC_rand, 0.13578566791465646)
-
-        self.assertAlmostEqual(c.EC_sim, 0.9980676303238214)
-        self.assertAlmostEqual(c.VC_sim, 0.034430408799858946)
-        self.assertAlmostEqual(c.p_sim, 0.001)
-        self.assertAlmostEqual(c.p_z_sim, 0.00016908100514811952)
-        self.assertAlmostEqual(c.z_sim, -3.5841621159171746)
-        self.assertAlmostEqual(c.seC_sim, 0.18555432843202269)
```

Comment:
This block of changes is much larger than that in the [pysal diff](https://github.com/pysal/pysal/pull/975/files#diff-f9b37298f2320cf626889ea09791bb21). is that just git magic? I don't see any differences in the numbers.